components/Column.js

```diff
@@ -82,7 +82,8 @@ export default class Column extends Component {
     return bricks.map((brick, index) => {
```

Comment:
@srameshr Can you add imageContainerStyle to the Columns.PropTypes? Here as well as the `Masonry` component.

Reply:
Sure! On it.

Reply:
Done